packages/gatsby/src/db/loki/nodes.js

```diff
@@ -204,6 +205,8 @@ function createNode(node, oldNode) {
   invariant(node.internal.type, `node has no "internal.type" field`)
   invariant(node.id, `node has no "id" field`)
 
+  trackCli(`CREATE_LOKI_NODE`, {}, { debounce: true })
```

Comment:
What's the reason for specifically track loki nodes? All node creation should be captured by https://github.com/gatsbyjs/gatsby/pull/14923/files#diff-c739d0ad63537276e8fc320522a8b985R594

Reply:
I'm not 100% sure if that's necessary as I only have partial understanding of this part. The main idea was to ensure we don't miss out anyting if/when we move between  redux and loki.